src/Sylius/Bundle/PromotionBundle/DependencyInjection/SyliusPromotionExtension.php

```diff
@@ -33,6 +33,9 @@ public function load(array $configs, ContainerBuilder $container): void
         $loader->load('services.xml');
         $loader->load(sprintf('services/integrations/%s.xml', $config['driver']));
 
+        $container->setParameter('sylius.promotion.promotion_action.validation_groups', $config['promotion_action']['validation_groups']);
```

Comment:
Is it possible to check configured actions/rules against action/rules registry, which is configured via compiler pass over here?